### ISSUE TITLE
numbers not centrally aligned

### ISSUE THREAD

**Issue by kikipluche** [swarmcity/sc-terminal#55]
It looks odd for me. But that is personal
<img width="255" alt="screen shot 2017-01-25 at 23 48 23" src="https://cloud.githubusercontent.com/assets/9036081/22312601/d0c2933c-e358-11e6-9739-06361fa0b2d0.png">

**Comment by faffydee** [comment]
Yeah, the balance generates a space behind it somehow. I actually don't know how to fix that.

**Comment by kikipluche** [comment]
sponnet, check space after balance
